add_nyt_review_link.py

```diff
@@ -114,11 +118,10 @@ def __process_found_bestseller_edition(self, bstslr_record_isbn,
                 'for the work {} of the edition {}'
                     .format(bstslr_edition.work.olid,
                             bstslr_record_isbn))
-            self.__add_link(work, link_struct)
-            work.save(comment)
```

Comment:
If we don't specify the comment here where does it get specified?

Reply:
I don't see a way to provide a comment if the save function is used. That's why I had to remove the comment altogether.

Reply:
Good point! I was not aware Python has closures. Modified the code.